### PR TITLE
data:ChinatuL

### PR DESCRIPTION
**Pull request by ChinatuL**
## Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4597"></a>

### CODE REVIEW

File: data/ChinatuL.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/chinatulucia/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/chinatu-lucia-3b923b1b1/",
+      "icon": "FaLinkedIn"
```
Comment (Panquesito7) [inline data/ChinatuL.json:20]: Wrong icon name.
```suggestion
      "icon": "FaLinkedin"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/ChinatuL.json (hunk 1 above)
Comment (ChinatuL) [reply data/ChinatuL.json:20]: Thank you very much for your review. I would effect that immediately

**Review by Hrishi5111998 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**